miden-lib/asm/scripts/ASWAP.masm

```diff
@@ -0,0 +1,52 @@
+use.miden::sat::note
+use.miden::wallets::basic->wallet
+
+# Atomic Swap script: adds an asset from the note into consumers account and
+# creates a note consumable by note issuer containing requested ASSET.
+#
+# Requires that the account exposes: 
+#
+# Inputs: [ASSET, serial_num]
+# Outputs: []
+#
+# Note inputs are assumed to be as follows:
+# - asset:
+# - serial_num
```

Comment:
I think note inputs here should be similar to what I mentioned in https://github.com/0xPolygonMiden/miden-base/pull/363#discussion_r1428488776. Specifically: `[RECIPIENT, ASSET, TAG]`, where `TAG = [tag, 0, 0, 0]` and `RECIPIENT` is computed as I described in one of the previous comments.

Reply:
Done